protocol/contracts/governance/ElderElection.sol

```diff
@@ -120,6 +137,10 @@ contract ElderElection is Ownable, EIP712 {
     }
 
     function _setEndorsements(address account, uint256[] memory discordIds) internal {
+        if (discordIds.length > numCandidates) {
+          revert TooManyEndorsements();
+        }
+
         endorsementsBy[account] = discordIds;
```

Comment:
Is it fair to say it's the responsibility of the caller to ensure `discordIds` are all valid - the contract won't check that `candidates[discordId] == true` for each of these ids (this could be a large list - so don't want to check that on chain)

Reply:
Yep - there's no harm in the discordIds being invalid. They just won't count for anything.